storybook/src/Twfiek/Molecules/TextDefault/TextDefault.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState, useRef } from "react";
+import AddOption from "./components/AddOption";
+import OptionInput from "./components/OptionInput";
+
+const TextDefault = () => {
+  const [addOption, setAddOption] = useState([{ id: 1 }, { id: 2 }]);
+  const [inputVal, setInputVal] = useState();
+  const inputParent = useRef(null);
+  // const alpha = "abcdefghijklmnopqrstuvwxyz".toUpperCase();
+  //  Alphabet letters
+  const letters = (() => {
+    const caps = [...Array(26)].map((val, i) => String.fromCharCode(i + 65));
+    return caps;
+  })();
+  const addOptionHandler = () => {
+    const numbers = "123456789";
+    const randId = Math.floor(Math.random() * numbers);
+    //alpha: alpha[id - 1]
+    setAddOption([...addOption, { id: randId }]);
+  };
+  const removeOptionHandler = (e) => {
+    setAddOption(
+      addOption.filter((item) => item.id.toString() !== e.target.id)
+    );
+  };
+  const inputValHandler = () => {
+    let inputValues = [];
+    inputParent.current.childNodes.forEach((input) => {
```

Comment:
not sure why you are doing this instead of lifting up the state with the needed values/methods?